drivers/net/ethernet/renesas/rswitch.h

```diff
@@ -306,6 +313,8 @@ struct rswitch_device {
 	 */
 	struct device *vlan_parent;
 	bool mondev;
+	struct rswitch_vmq_status *vmq_info;
+	bool is_vmq;
```

Comment:
I believe it is a good time to introduce some `enum rdev_type`. Because we already have `is_vmq`and `mondev` fields, implicit checks for being vlan or  front device... This is getting out of hand.